pkg/inject/report.go

```diff
@@ -68,3 +83,35 @@ func checkUDPPorts(t *v1.PodSpec) bool {
 	}
 	return false
 }
+
+func (r *Report) disableByAnnotation(conf *ResourceConfig) bool {
+	// truth table of the effects of the inject annotation on the webhook:
+	//
+	// namespace | pod      | inject?  | return
+	// --------- | -------- | -------- | ------
+	// enabled   | enabled  | yes      | false
+	// enabled   | ""       | yes      | false
+	// enabled   | disabled | no       | true
+	// disabled  | enabled  | yes      | false
+	// ""        | enabled  | yes      | false
+	// disabled  | disabled | no       | true
+	// ""        | disabled | no       | true
+	// disabled  | ""       | no       | true
+	// ""        | ""       | no       | true
+	//
+	// for CLI, only the 'disabled' annotation is taken into consideration
+	// for its opt-out effect
```

Comment:
to confirm my own understanding, and for others going forward, can you verify this updated table is correct, and then include it?
```
	// truth table of the effects of the inject annotation on the webhook and cli:
	//
	// origin  | namespace | pod      | inject?  | return
	// ------- | --------- | -------- | -------- | ------
	// webhook | enabled   | enabled  | yes      | false
	// webhook | enabled   | ""       | yes      | false
	// webhook | enabled   | disabled | no       | true
	// webhook | disabled  | enabled  | yes      | false
	// webhook | ""        | enabled  | yes      | false
	// webhook | disabled  | disabled | no       | true
	// webhook | ""        | disabled | no       | true
	// webhook | disabled  | ""       | no       | true
	// webhook | ""        | ""       | no       | true
	// cli     | n/a       | enabled  | yes      | false
	// cli     | n/a       | ""       | yes      | false
	// cli     | n/a       | disabled | no       | true
	//
	// for CLI, only the 'disabled' annotation is taken into consideration
	// for its opt-out effect
```

Reply:
Thanks @siggy. That extra column makes it even clearer :+1:. And LGTM.